gson/src/main/java/com/google/gson/reflect/impl/UnsafeReflectionAccessor.java

```diff
@@ -27,7 +27,7 @@
  * NOTE: This implementation is designed for Java 9. Although it should work with earlier Java releases, it is better to
  * use {@link PreJava9ReflectionAccessor} for them.
  */
-public class UnsafeReflectionAccessor implements ReflectionAccessor {
+public final class UnsafeReflectionAccessor extends ReflectionAccessor {
```

Comment:
package private?

Reply:
It implies that 'reflect.impl' classes need to be moved up to the 'reflect' package. Seems fine after moving to a separate 'internal' package though...